web/admin/src/utils/drag.ts

```diff
@@ -200,3 +200,19 @@ export function getSiblingItemIds(
 
   return result;
 }
+
+export const collapseAllFolders = (
```

Comment:
在drag.ts文件中，新添加的collapseAllFolders函数可以添加注释以提高可读性。

```suggestion
/**
 * 折叠/展开所有文件夹
 * @param list 树形结构列表
 * @param collapsed 是否折叠
 * @returns 折叠/展开后的树形结构列表
 */
export const collapseAllFolders = (
  list: TreeItems<ITreeItem>,
  collapsed: boolean,
): TreeItems<ITreeItem> => {
  return list.map(it => ({
    ...it,
    collapsed: it.type === 1 ? collapsed : it.collapsed,
    children: it.children
      ? (collapseAllFolders(
          it.children as TreeItems<ITreeItem>,
          collapsed,
        ) as ITreeItem[])
      : it.children,
  }));
};